protocols/v2/noise-sv2/src/handshake.rs

```diff
@@ -1,3 +1,32 @@
+// # Noise Protocol Handshake Operations
```

Comment:
I think it should be mentioned that the Handshake process happens only once and explain how the state is saved for future messages, for me it is not very clear how that is state is maintained and how it is accessed in future interaction

Reply:
Shouldn't this be reviewed according to the specs?

Reply:
I do think we should explain how things work in practice